### DISCUSSION TITLE
Is it possible to delete other users' messages?

### DISCUSSION THREAD

**Discussion by SBR0593** [victornpb/undiscord#551] [1. Questions & Answers]
If there is a situation in which I need to or want to delete another user's message, would that be possible for me to do so, and if it is possible would that lead to Discord banning me even if it's only one to three messages?

**Comment by avizmarlon** [comment]
In DMs you can't.

In servers, you need "Manage Messages" permission or permission to control a bot that is able to "Manage Messages".
